src/classroom.rb

```diff
@@ -0,0 +1,18 @@
+# Create a class Classroom
+class Classroom
+  # Setter and getter for @label
+  attr_accessor :label
+  attr_reader :students
+
+  # @label instance variable, should be initialized in the constructor.
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  #   Create the has-many side (a classroom has many students).
+  def add_student(student)
+    @students.push(student) unless @students.include?(student)
+    student.classroom = self
+  end
+end
```

Comment:
- [ **OPTIONAL** ] In `README.md` **file:** As pointed out the previous reviewer, kindly make sure the you have added the ` Future Features` section in your README and this should include **1-3 potential features** to be added to your application.
Good :+1: job :100: adding other changes required by the previous reviewer.

Reply:
I have these in the features section. What the project would look like at the end of the project,

Reply:
Unless it should be something different.